Slim/Formats/Movie.pm

```diff
@@ -252,13 +252,13 @@ sub parseStream {
 	my $offset = $args->{_offset};
 	my $log = logger('player.streaming');
 
-	while (length($args->{_scanbuf}) > $args->{_offset} + $args->{_need} + 8) {
```

Comment:
Is this just an optimization, which shouldn't have any impact?...

Reply:
Kind of - using $offset is just easier to read